demo/crud-advanced-demos.html

```diff
@@ -13,7 +13,8 @@ <h3>Custom search and renderers</h3>
     <vaadin-demo-snippet id="crud-advanced-demos-custom-search">
       <template preserve-content>
         <vaadin-crud  no-head theme="no-border compact" style="flex-direction: column-reverse">
-          <vaadin-text-field slot="footer" style="width: calc(100% - 0.5em)" placeholder="Search users by first or last name">
+          <vaadin-button new-button slot="toolbar" style="margin-left: 0.5em">New user</vaadin-button>
+          <vaadin-text-field slot="toolbar" style="flex: 1" placeholder="Search users by first or last name">
```

Comment:
Nit: would be great if we could avoid margins in inline styles (that the toolbar theme would add default margins for the slotted content. Not blocking this PR, but could be fixed at some point (make it easier to add multiple buttons in the toolbar for example).